response_operations_ui/controllers/message_controllers.py

```diff
@@ -39,6 +39,10 @@ def get_conversation(thread_id):
 def get_conversation_count(params):
     logger.info("Retrieving count of threads")
 
+    params['is_closed'] = 'true' if params['is_closed'] else 'false'
```

Comment:
Does the if work if it's missing?  If there isn't a 'is_closed' key in the dictionary then it'll throw a KeyError.

Reply:
Will change to get with default